images/newrelic/tests/main.tf

```diff
@@ -0,0 +1,115 @@
+terraform {
+  required_providers {
+    oci  = { source = "chainguard-dev/oci" }
+    helm = { source = "hashicorp/helm" }
+  }
+}
+
+variable "digests" {
+  description = "The image digests to run tests over."
+  type = object({
+    infrastructure-bundle   = string
+    fluent-bit-output       = string
+    k8s-events-forwarder    = string
+    prometheus-configurator = string
+    kube-events             = string
+    prometheus              = string
+    kubernetes              = string
+  })
+}
+
+data "oci_string" "ref" {
+  for_each = var.digests
+  input    = each.value
+}
+
+# This is a pretty cursed method of getting the license key directly from the
+# environment, without needing to plumb through TF_VAR_
+data "external" "license-key" {
+  program = ["bash", "-c", "echo \"{\\\"output\\\": \\\"$NEW_RELIC_LICENSE_KEY\\\"}\""]
+}
```

Comment:
Wow. I'm not even mad, I'm impressed.

We should definitely drop this, but in the meantime you've taught me a new cursed way to pipe things through, and I'll never forgive you. 😁